setup.py

```diff
@@ -1,16 +1,7 @@
 #!/usr/bin/env python
 # encoding: utf-8
 
-# n.b. we can't have unicode_literals here due to http://bugs.python.org/setuptools/issue152
-from __future__ import absolute_import, division, print_function
```

Comment:
We should leave `from __future__ import absolute_import` in here until we drop Python 2 support.

Reply:
Nothing in setup.py actually needs that import, as far as I can see?